src/main/java/uk/gov/ons/ctp/integration/ratelimiter/client/RateLimiterClient.java

```diff
@@ -19,22 +20,28 @@ public RateLimiterClient(RestClient rateLimiterClient) {
     this.rateLimiterClient = rateLimiterClient;
   }
 
-  public RateLimitResponse checkRateLimit(RateLimitRequest rateLimitRequest) {
-    log.with("domain", rateLimitRequest.getDomain())
+  public RateLimitResponse checkRateLimit(String domain, Product product,
```

Comment:
Please make the domain a public enum that the caller elects ie RH("respondenthome") then knowledge of the exact string is kept within the client lib, not the caller.